File: M3/Practica3/Exercici1.java
import java.io.*; 
// Programa que calcula la raiz quadrada
public class Exercici1 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
		System.out.print("Introduce un numero positivo=");
        String numero = reader.readLine();
        float A = Float.parseFloat(numero);
        while (A<0){
			System.out.println("Error no puede ser un numero negativo");
			System.out.print("Introduce un numero positivo=");
			String numero2 = reader.readLine();
			float C = Float.parseFloat(numero2);
			A=C;	
		}
		float B=(float) Math.sqrt(A);
		System.out.println("La raiz cuadrada es="+B);		
    }
}

File: M3/Practica3/Exercici1.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
# Programa que calcula la raiz quadrada
import math
A=float(raw_input("Introduce un numero positivo="))
while A<0:
	print "Error no puede ser un numero negativo"
	A=float(raw_input("Introduce un numero positivo="))
A=math.sqrt(A)
print "La raiz cuadrada es=", A
